src/components/cc-tile-metrics/cc-tile-metrics.js

```diff
@@ -143,6 +156,11 @@ export class CcTileMetrics extends LitElement {
     });
   }
 
+  /**
+   * @param {Metric[]} inputData
+   * @returns {ChartData}
+   * @private
+   */
   _getChartData (inputData) {
```

Comment:
fix: add `skeleton` argument to this function and use it instead of the `skeleton` property of the inputData

Reply:
This is a lot nicer indeed, thanks! :sunglasses: